cmd/fleet/delete_fleet.go

```diff
@@ -0,0 +1,59 @@
+package fleet
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/calyptia/cli/completer"
+	"github.com/calyptia/cli/config"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdDeleteFleet(config *config.Config) *cobra.Command {
+	var confirmed bool
+	completer := completer.Completer{Config: config}
+
+	cmd := &cobra.Command{
+		Use:               "fleet FLEET",
+		Short:             "delete a fleet by ID or name",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: completer.CompleteFleets,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fleetKey := args[0]
+			if !confirmed {
+				cmd.Printf("Are yo sure you want to delete %q? (y/N)", fleetKey)
+				var answer string
+				_, err := fmt.Scanln(&answer)
```

Comment:
I think there was an utility to read confirmation...

Reply:
```go
confirmed, err := confirm.Read(cmd.InOrStdin())
if err != nil {
	return err
}

if !confirmed {
	cmd.Println("Aborted")
	return nil
}
```

Reply:
Implemented! Thanks.